### ISSUE TITLE
Bring 2.4 release on fork to this repo

### ISSUE THREAD

**Issue by alerque** [theleagueof/league-spartan#33]
@sursly tagged a `2.4` [release on his fork](https://github.com/sursly/league-spartan/releases/tag/2.4), but it is based on the version of the repo before I added Fontship release stuff here. The changes in there need to be cherry picked or rebased into the current repository and a `2.400` release tagged.

(Tyler if you start new work in up to date branches from the latest upstream version instead of your old ones this wouldn't need doing, wink wink. I can take care of this, but it would be a lot smoother if you just could open a PR to submit new work and then tag here and be done with the release process.)

**Comment by davelab6** [comment]
Was the work done for GF-ization merged into either or both of these branches? :)

**Comment by davelab6** [comment]
Perhaps we should set up a video call with all the possible contributors

@sursly @alerque @bghryct @MattBaileyDesign @aaronbell

**Comment by bghryct** [comment]
@davelab6 I don't know that @MattBaileyDesign version of Spartan is similar enough to the LOMT version of Spartan to be lumped together. Happy to be part of a video call though.

**Comment by MattBaileyDesign** [comment]
@bghryzt I'm also a little unsure of this, but I'm also happy to hop on a call. Right now I have a WIP version of Spartan MB with obliques that may be of interest.

**Comment by alerque** [comment]
I dropped this comment on issue #32 before I saw this thread.

> I'm sorry, but I don't see the connection between the *Spartan MB* on Google Fonts and the *League Spartan* seen here. Very similar, but they appear at first blush to be completely parallel efforts (something Google has become good at sponsoring). If that first quick research is wrong then at the very least there is a huge documentation and credit gap because neither project credits the other in any way, including as would be required by licensing.

I'd be happy to join a call on this issue and can hopefully help sort out the technical threads. But there is also a thread of *design* work that I can't speak to as well, not to mention personal vested interest and IP. Definitely @sursly and/or @micahbrich need to be involved in this to speak to those angles. I know some of the current LOMT technical situations and can help wrangle the merging, but only after others decide what is supposed to happen.

**Comment by davelab6** [comment]
> they appear at first blush to be completely parallel efforts 

Ah, okay great - sorry I got confused about that. 

In this case, I guess GF would onboard "League Spartan" as a separate family.

**Comment by sursly** [comment]
@davelab6 My latest version(s) of League Spartan plays pretty well with the GF standards. I'm not volunteering @RosaWagner for anything, but she's handled my source files and build scripts incredibly well so far.

**Comment by davelab6** [comment]
Sweet! Happy to queue up a GF release of "League Spartan", although for Q4
the onboarding folks are turning towards process/docs/tools improvements so
that ideally we can pick up the pace of updates and new releases even
faster next year - so, I leave it with Rosalie how to slot this into the
release stack :)

**Comment by micahbrich** [comment]
> The RFN is making sure that only this version of the font has this name, and modified versions in another fork need to have another name. The problem is that GF's API is going to subset the font etc which means it won't be this exact version that is going to be served. So If there is a RFN, then we have to publish the font under another name. Exception can be made but we try to avoid it.
> 
> Trademark is not an issue.

Awesome you folks are working on this. 👏 What do you need, how can I help? I’ma little unclear.
